GaugeChart.vue

```diff
@@ -105,6 +96,18 @@ export default {
         }
       }
     });
+  },
+  methods: {
+    setChartData(latest) {
+      if(!latest || !this.chart) {
+        return false;
+      }
+      this.last = Number(latest).toFixed(2);
+      const [values] = this.chart.data.datasets;
+      const next = [this.last, (1 - this.last).toFixed(2)];
```

Comment:
is this intermediate variable needed?  Replace with a comment if required for clarity.

Reply:
I went ahead and turned this into a function and removed the next. I do think it added clarity, but I think it was primarily a carry over from a previous revision.